<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>GSSControlStation</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/files/images_icons/apps/globe.ico</normaloff>:/files/images_icons/apps/globe.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>40</width>
    <height>40</height>
   </size>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Settings"/>
    <addaction name="actionSave_Settings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCommunication">
    <property name="title">
     <string>Communication</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menuPerspectives">
    <property name="title">
     <string>Perspectives</string>
    </property>
    <addaction name="actionOperator"/>
    <addaction name="actionEngineer"/>
   </widget>
   <widget class="QMenu" name="menuMain_Widgits">
    <property name="title">
     <string>Main Widgets</string>
    </property>
    <addaction name="actionSystem_configuration"/>
   </widget>
   <widget class="QMenu" name="menuTool_Widgets">
    <property name="title">
     <string>Tool Widgets</string>
    </property>
    <addaction name="actionDebug_Console"/>
    <addaction name="actionParameter"/>
    <addaction name="actionSystem_Status"/>
   </widget>
   <widget class="QMenu" name="menuHelps">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_GSSControlStation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCommunication"/>
   <addaction name="menuPerspectives"/>
   <addaction name="menuMain_Widgits"/>
   <addaction name="menuTool_Widgets"/>
   <addaction name="menuHelps"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Settings">
   <property name="icon">
    <iconset>
     <normaloff>:/files/images_icons/apps/download.ico</normaloff>:/files/images_icons/apps/download.ico</iconset>
   </property>
   <property name="text">
    <string>Load Settings</string>
   </property>
  </action>
  <action name="actionSave_Settings">
   <property name="icon">
    <iconset>
     <normaloff>:/files/images_icons/apps/save.ico</normaloff>:/files/images_icons/apps/save.ico</iconset>
   </property>
   <property name="text">
    <string>Save Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>:/files/images_icons/apps/button_cancel_256.ico</normaloff>:/files/images_icons/apps/button_cancel_256.ico</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset>
     <normaloff>:/files/images_icons/apps/connect.ico</normaloff>:/files/images_icons/apps/connect.ico</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionOperator">
   <property name="icon">
    <iconset>
     <normaloff>:/files/images_icons/apps/system-users.svg</normaloff>:/files/images_icons/apps/system-users.svg</iconset>
   </property>
   <property name="text">
    <string>Operator</string>
   </property>
  </action>
  <action name="actionEngineer">
   <property name="icon">
    <iconset>
     <normaloff>:/files/images_icons/apps/engineer_avatar.ico</normaloff>:/files/images_icons/apps/engineer_avatar.ico</iconset>
   </property>
   <property name="text">
    <string>Engineer</string>
   </property>
  </action>
  <action name="actionSystem_configuration">
   <property name="icon">
    <iconset>
     <normaloff>:/files/images_icons/apps/preferences_system.ico</normaloff>:/files/images_icons/apps/preferences_system.ico</iconset>
   </property>
   <property name="text">
    <string>System Configuration</string>
   </property>
  </action>
  <action name="actionDebug_Console">
   <property name="icon">
    <iconset>
     <normaloff>:/files/images_icons/apps/guake.ico</normaloff>:/files/images_icons/apps/guake.ico</iconset>
   </property>
   <property name="text">
    <string>Debug Console</string>
   </property>
  </action>
  <action name="actionParameter">
   <property name="text">
    <string>Parameter</string>
   </property>
  </action>
  <action name="actionSystem_Status">
   <property name="text">
    <string>System Status</string>
   </property>
  </action>
  <action name="actionAbout_GSSControlStation">
   <property name="icon">
    <iconset>
     <normaloff>:/files/images_icons/apps/info_box_blue.ico</normaloff>:/files/images_icons/apps/info_box_blue.ico</iconset>
   </property>
   <property name="text">
    <string>About GSSControlStation</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset>
     <normaloff>:/files/images_icons/apps/disconnect.ico</normaloff>:/files/images_icons/apps/disconnect.ico</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
